README.md

```diff
@@ -30,6 +30,7 @@ A Project which includes curated list of different login page web pages.
 | login-form-v10 | <img src="https://raw.githubusercontent.com/LoginRadius/awesome-login-pages/main/login-form-v10/login-form-v10/screenshot.png " width="480" > | [@abhir9](https://github.com/abhir9)               |
 | login-form-v11 | <img src="https://raw.githubusercontent.com/LoginRadius/awesome-login-pages/main/login-form-v11/login-form-v11/screenshot.png " width="480" > | [@abhir9](https://github.com/abhir9)               |
 | login-form-v12 | <img src="https://raw.githubusercontent.com/LoginRadius/awesome-login-pages/main/login-form-v12/login-form-v12/screenshot.png " width="480" > | [@abhir9](https://github.com/abhir9)               |
+| login-Form-ver1| <img src="https://raw.githubusercontent.com/pandeyom331/awesome-login-pages/main/login-Form-ver1/ss.PNG" width="480" > |   [@pandeyom331](https://github.com/pandeyom331)          |
```

Comment:
@pandeyom331 
please update Loginradius Path instead own repo path

Reply:
rename the image to screenshot.png

Reply:
loginradius path means ?

Reply:
LoginRadius/awesome-login-pages/main/login-form-v12/login-form-v12/screenshot.png

pandeyom331/awesome-login-pages/main/login-Form-ver1/ss.PNG

it images will stay in main repo even if forked repo remove the resource

Reply:
ok thanks, I understood, I'll do it now.